app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -267,29 +269,37 @@ export class FieldErrorHint extends Component {
       staticHint,
       widthContent,
       darkView,
+      provideHint,
       ...rest
     } = this.props;
     const classes = cx('field-error-hint', `type-${hintType}`);
 
     return (
       <div className={classes}>
-        {children && cloneElement(children, { error, active, ...rest })}
-        <div
-          className={cx('hint', `type-${hintType}`, {
-            'static-hint': staticHint,
-            visible: this.isHintVisible(),
+        {children &&
+          cloneElement(children, {
+            error: error && messages[error] ? intl.formatMessage(messages[error]) : error,
+            active,
+            ...rest,
           })}
-        >
+        {provideHint && (
```

Comment:
I guess, it is a temporary solution for omit hint(bar which appear while error exist) because right now all errors text store in this component. So this prop omit hint and FieldErrorHint work as a proxy between reduxForm and any component which needs

Reply:
May be we don't need this component at all for the new fields? Each field now is responsible to show an error.

Reply:
In general yes, but now validator return error key not error message so we should decide it's ok to see `requiredFiled` but not human readable message like `field shouldn't be empty`

Reply:
Ok, got it. We can create another task to adjust the behavior of errors.